upload-charm/action.yaml

```diff
@@ -37,6 +37,13 @@ inputs:
     default: 'true'
     description: |
       Set to false if you don't want to update the OCI image
+  local-image:
+    required: false
+    default: 'false'
+    description: |
+      Set to true if you want to use local OCI images. They must
+      be built prior to running the action. Set the upstream-resource
```

Comment:
Isn't the adopted keyword `upstream-source`? https://juju.is/docs/sdk/metadata-yaml#heading--other-arbitrary-keywords

Reply:
Good spot, fixed.